### ISSUE TITLE
sbt-coursier errors with sbt-dotty

### ISSUE THREAD

**Issue by alexarchambault** [coursier/coursier#1087]
It seems the scala organization rewrite is too aggressive since sbt-coursier `1.1.0-M12` (`1.1.0-M11` is fine), which makes resolution fails.

With just sbt-coursier, dotty still uses ivy via `sbt.Keys.dependencyResolution`, to fetch some of its JARs. sbt-lm-coursier provides a `dependencyResolution`, but fails for similar reasons (during `scalaInstance` rather than `update`).

**Comment by alexarchambault** [comment]
Error with sbt-coursier:

```bash
$ git clone https://github.com/lampepfl/dotty-example-project.git # a753b14 as of writing this
$ cd dotty-example-project
$ mkdir -p project/project
$ cat > project/project/plugins.sbt << EOF
addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M12")
EOF
$ echo "addSbtCoursier" >> project/plugins.sbt
$ sbt run
[info] Loading settings for project global-plugins from idea.sbt,plugins.sbt,metals.sbt ...
[info] Loading global plugins from /Users/alexandre/.sbt/1.0/plugins
[info] Loading settings for project dotty-example-project-build-build from plugins.sbt ...
[info] Loading project definition from /Users/alexandre/projects/other/dotty-example-project/project/project
[info] Updating ProjectRef(uri("file:/Users/alexandre/projects/other/dotty-example-project/project/project/"), "dotty-example-project-build-build")...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Loading settings for project dotty-example-project-build from plugins.sbt ...
[info] Loading project definition from /Users/alexandre/projects/other/dotty-example-project/project
[info] Loading settings for project root from build.sbt ...
[info] Set current project to dotty-example-project (in build file:/Users/alexandre/projects/other/dotty-example-project/)
[info] Updating root
[info] Resolved root dependencies
[error] coursier.error.ResolutionError$CantDownloadModule: Error downloading org.scala-lang:scala-library:0.13.0-RC1
[error]   Not found
[error]   not found: /Users/alexandre/.ivy2/local/org.scala-lang/scala-library/0.13.0-RC1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-lang/scala-library/0.13.0-RC1/scala-library-0.13.0-RC1.pom
[error] 	at coursier.Resolve$.$anonfun$validate$1(Resolve.scala:258)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:233)
[error] 	at scala.collection.Iterator.foreach(Iterator.scala:937)
[error] 	at scala.collection.Iterator.foreach$(Iterator.scala:937)
[error] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
[error] 	at scala.collection.IterableLike.foreach(IterableLike.scala:70)
[error] 	at scala.collection.IterableLike.foreach$(IterableLike.scala:69)
[error] 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:233)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:226)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at coursier.Resolve$.validate(Resolve.scala:256)
[error] 	at coursier.Resolve.validate0$1(Resolve.scala:73)
[error] 	at coursier.Resolve.$anonfun$ioWithConflicts$4(Resolve.scala:117)
[error] 	at coursier.util.Task$.$anonfun$flatMap$2(Task.scala:11)
[error] 	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:303)
[error] 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
[error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error] 	at java.lang.Thread.run(Thread.java:745)
[error] (coursierResolutions) coursier.error.ResolutionError$CantDownloadModule: Error downloading org.scala-lang:scala-library:0.13.0-RC1
[error]   Not found
[error]   not found: /Users/alexandre/.ivy2/local/org.scala-lang/scala-library/0.13.0-RC1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scala-lang/scala-library/0.13.0-RC1/scala-library-0.13.0-RC1.pom
[error] Total time: 2 s, completed 6 mars 2019 10:12:35
```

**Comment by alexarchambault** [comment]
Changing sbt-coursier to sbt-lm-coursier:
```
[info] Loading settings for project global-plugins from idea.sbt,plugins.sbt,metals.sbt ...
[info] Loading global plugins from /Users/alexandre/.sbt/1.0/plugins
[info] Loading settings for project dotty-example-project-build-build from plugins.sbt ...
[info] Loading project definition from /Users/alexandre/projects/other/dotty-example-project/project/project
[info] Updating ProjectRef(uri("file:/Users/alexandre/projects/other/dotty-example-project/project/project/"), "dotty-example-project-build-build")...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Loading settings for project dotty-example-project-build from plugins.sbt ...
[info] Loading project definition from /Users/alexandre/projects/other/dotty-example-project/project
[info] Loading settings for project root from build.sbt ...
[info] Set current project to dotty-example-project (in build file:/Users/alexandre/projects/other/dotty-example-project/)
[info] Updating
[info] Resolved  dependencies
[error] Couldn't retrieve `ch.epfl.lamp:dotty-doc:0.13.0-RC1`.
[error] (scalaInstance) Couldn't retrieve `ch.epfl.lamp:dotty-doc:0.13.0-RC1`.
[error] Total time: 1 s, completed 6 mars 2019 10:16:17
```

**Comment by alexarchambault** [comment]
Should be fixed by https://github.com/coursier/sbt-coursier/pull/42 (to be included in the upcoming sbt-coursier `1.1.0-M13`).
